app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -106,6 +107,20 @@ def check_enrollment(enrollment)
     status_check_attempted_at = Time.zone.now
     enrollment_outcomes[:enrollments_checked] += 1
 
+    profile_deactivation_reason = enrollment.profile_deactivation_reason
+
+    if profile_deactivation_reason.present?
+      log_enrollment_updated_analytics(
+        enrollment: enrollment,
+        enrollment_passed: false,
+        enrollment_completed: true,
+        response: nil,
+        reason: "Profile has a deactivation reason of #{profile_deactivation_reason}",
+      )
+      cancel_enrollment(enrollment)
+      return
```

Comment:
nit: do you need the `return` here?

Reply:
The return is needed here because the enrollment no longer needs to be processed when the profile has a deactivation reason.